src/components/sbb-selection-panel/sbb-selection-panel.tsx

```diff
@@ -185,49 +161,56 @@ export class SbbSelectionPanel implements ComponentInterface {
 
   private _handleOpening(): void {
     this._state = 'opened';
-    this.didOpen.emit();
+    this._didOpen.emit();
   }
 
   private _handleClosing(): void {
     this._state = 'closed';
-    this.didClose.emit();
+    this._didClose.emit();
   }
 
-  public render(): JSX.Element {
-    return (
-      <Host
-        data-has-content={this._namedSlots['content']}
-        data-state={this._state}
-        data-checked={this._checked}
-        data-disabled={this._disabled}
-      >
-        <div class="sbb-selection-panel">
-          <div class="sbb-selection-panel__badge">
-            <slot name="badge" />
-          </div>
-
-          <div class="sbb-selection-panel__input">
-            <slot />
-          </div>
-
-          {this._namedSlots['content'] && (
-            <div
+  protected override render(): TemplateResult {
+    setAttribute(this, 'data-has-content', this._namedSlots['content']);
+    setAttribute(this, 'data-state', this._state);
+    setAttribute(this, 'data-checked', this._checked);
+    setAttribute(this, 'data-disabled', this._disabled);
+
+    return html`
+      <div class="sbb-selection-panel">
+        <div class="sbb-selection-panel__badge">
+          <slot name="badge"></slot>
+        </div>
+
+        <div class="sbb-selection-panel__input">
+          <slot></slot>
+        </div>
+
+        ${this._namedSlots['content']
+          ? html` <div
               class="sbb-selection-panel__content--wrapper"
-              data-expanded={this._checked || this.forceOpen}
-              ref={(el) => {
+              data-expanded=${this._checked || this.forceOpen}
+              ${ref((el: HTMLElement) => {
                 this._contentElement = el;
-                this._contentElement.inert = !this._checked && !this.forceOpen;
-              }}
-              onTransitionEnd={(event: TransitionEvent) => this._onTransitionEnd(event)}
+                if (this._contentElement) {
+                  this._contentElement.inert = !this._checked && !this.forceOpen;
+                }
+              })}
+              @transitionend=${(event: TransitionEvent) => this._onTransitionEnd(event)}
             >
               <div class="sbb-selection-panel__content">
-                <sbb-divider />
-                <slot name="content" />
+                <sbb-divider></sbb-divider>
```

Comment:
You should also add `import '../sbb-divider'` since you are using it inside the component

Reply:
Done, and test fixed